src/components/chart/ha-chart-base.ts

```diff
@@ -316,6 +316,15 @@ export class HaChartBase extends LitElement {
           }
         });
       }
+
+      Object.entries(this.options?.legend?.selected || {}).forEach(
```

Comment:
hmm I thought this worked in my local. 😕 

Maybe needs to be passed some other way.